packages/web/webpack/server.ts

```diff
@@ -14,7 +14,7 @@ export const serverConfig: (options: WebpackConfigOptions) => Configuration = ({
 }) => ({
   entry: join(__dirname, '..', srcDir, 'server.tsx'),
   ...configCommon(mode),
-  devtool: mode === 'production' ? false : 'inline-cheap-module-source-map',
+  devtool: 'inline-cheap-module-source-map',
```

Comment:
Could you please clarify the reasoning behind this change?

Reply:
TODO: replace inlining with separate sourcemap file